src/flatc.cpp

```diff
@@ -130,6 +130,7 @@ std::string FlatCompiler::GetUsageString(const char *program_name) const {
     "  --keep-prefix      Keep original prefix of schema include statement.\n"
     "  --no-fb-import     Don't include flatbuffers import statement for TypeScript.\n"
     "  --no-ts-reexport   Don't re-export imported dependencies for TypeScript.\n"
+    "  --short-names      Use short function names for JS and TypeScript.\n"
```

Comment:
Please add this to the docs as well.

Reply:
Sure, I also include missing flags whcih are not in the doc. 3min